### PR TITLE
Saving the fb users accesstokens and refresh tokens to nodebb user fi…

### PR DESCRIPTION
**Pull request by jongarrison**
I've added some minor functionality to save the accesstoken and refreshtoken from FB for the use of other parts of the application and thought maybe this would be a useful addition for others.

-Jon

### CODE REVIEW

File: library.js [hunk 1]
```diff
@@ -120,7 +120,17 @@
 		})
 	};
 
-	Facebook.login = function(fbid, name, email, picture, callback) {
+	Facebook.storeTokens = function(uid, accessToken, refreshToken) {
+		//JG: Actually save the useful stuff
+		winston.info("Storing received fb access information for uid(" + uid + ") accessToken(" + accessToken + ") refreshToken(" + refreshToken + ")");
+		user.setUserField(uid, 'fbaccesstoken', accessToken);
+		user.setUserField(uid, 'fbrefreshtoken', refreshToken);
```
Comment (julianlam) [inline library.js]: Minor nitpick, can you use `fb:accesstoken` and `fb:refreshtoken`?

**Comment by julianlam** [comment]
Ok to merge once fixed :y:

**Comment by jongarrison** [comment]
Sorry, just getting to this now, I'd be happy to make that change, but on line 65 you use "fbid" to store the facebook id, do want fbaccesstoken and fbrefreshtoken to be different?  Your preference, I'm happy to do it either way.
https://github.com/jongarrison/nodebb-plugin-sso-facebook/blob/master/library.js#L65

**Comment by julianlam** [comment]
Oh, then it's fine as is :+1:

**Comment by julianlam** [comment]
v1.1.6

**Comment by jongarrison** [comment]
Oh, cool.  Great!

**Pull request merged**